src/current/v23.2/ui-cdc-dashboard.md

```diff
@@ -9,65 +9,133 @@ The **Changefeeds** dashboard in the DB Console lets you monitor the [changefeed
 
-<img src="{{ 'images/v23.2/ui_sink_byte_traffic.png' | relative_url }}" alt="DB Console Sink Byte Traffic graph" style="border:1px solid #eee;max-width:100%" />
+<img src="{{ 'images/v23.2/ui-emitted-bytes.png' | relative_url }}" alt="DB Console Emitted Bytes Graph showing the time and emitted bites" style="border:1px solid #eee;max-width:100%" />
 
 Metric | Description
 --------|----
-**Emitted Bytes** | The number of bytes emitted by CockroachDB into the sink for changefeeds.
+**Emitted Bytes** | The number of bytes emitted by CockroachDB into the configured changefeed sinks.
 
 ## Sink Counts
 
-This graph shows:
+This graph displays data relating to the number of messages and flushes at the changefeed sink.
 
 - The number of messages that CockroachDB sent to the sink.
 - The number of flushes that the sink performed for changefeeds.
 
-<img src="{{ 'images/v23.2/ui_sink_counts.png' | relative_url }}" alt="DB Console Sink Counts graph" style="border:1px solid #eee;max-width:100%" />
+<img src="{{ 'images/v23.2/ui-sink-counts.png' | relative_url }}" alt="DB Console Sink Counts graph" style="border:1px solid #eee;max-width:100%" />
 
 Metric | Description
 --------|----
-**Messages** | The number of messages that CockroachDB sent to the sink for changefeeds.
-**Flushes** | The the number of flushes that the sink performed for changefeeds.
+**Messages** | The number of messages that CockroachDB sent to the downstream sink.
+**Flushes** | The total number of flushes to the sink.
 
-## Sink Timings
+## Max Checkpoint Latency
 
-This graph shows:
+This graph displays the most any changefeed's persisted [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}) is behind the present time. Larger values indicate issues with successfully ingesting or emitting changes. If errors cause a changefeed to restart, or the changefeed is [paused]({% link {{ page.version.version }}/pause-job.md %}) and unpaused, emitted data up to the last checkpoint may be re-emitted.
 
-- The time in milliseconds per second required by CockroachDB to send messages to the sink.
-- The time CockroachDB spent waiting for the sink to flush the messages for changefeeds.
+<img src="{{ 'images/v23.2/ui-max-checkpoint-latency.png' | relative_url }}" alt="DB Console Max Checkpoint Latency graph" style="border:1px solid #eee;max-width:100%" />
 
-<img src="{{ 'images/v23.2/ui_sink_timings.png' | relative_url }}" alt="DB Console Sink Timings graph" style="border:1px solid #eee;max-width:100%" />
+{{site.data.alerts.callout_info}}
+The maximum checkpoint latency is distinct from, and slower than, the commit-to-emit latency for individual change messages. For more information about resolved timestamps, see [Ordering guarantees]({% link {{ page.version.version }}/changefeed-messages.md %}#ordering-guarantees).
```

Comment:
Is "commit-to-emit latency" the same as "commit latency" in the above graph? If so, for consistency, maybe use "commit latency".